students/ShujinCao/lesson05/assignment/test_database.py

```diff
@@ -0,0 +1,16 @@
+import database as db
+import pytest
+
+def test_import():
+    results = db.list_available_products()
+    assert [*results] ==['prd001', 'prd003', 'prd004', 'prd005', 'prd006', 'prd008',
+                  'prd010']
+
+def test_list_renter_info():
+    results = db.list_renter_info("prd005")
+    assert [*results] == ['user001', 'user003']
+
+
+def test_list_available_products():
+    results = db.list_available_products()
+    assert [k for k in results.values()][4]['description'] == 'Portable heater'
```

Comment:
Nice start to testing. It is good to have tests not dependent on each other. There is no guarantee that tests are going to be run in a particular order, and it makes it so you can run a single test, which can be very helpful. Fixture can help with this. 
http://doc.pytest.org/en/latest/fixture.html